lib/src/easy_admob/easy_admob_banner_ad.dart

```diff
@@ -64,14 +63,14 @@ class EasyAdmobBannerAd extends EasyAdBase {
   dynamic show() {
     final ad = _bannerAd;
     if (ad == null) {
-      print('Warning: attempt to show rewarded before loaded.');
       return const SizedBox();
     }
 
-    return SizedBox(
+    return Container(
+      alignment: Alignment.center,
       child: AdWidget(ad: ad),
-      height: 50,
-      width: double.infinity,
+      height: adSize.height.toDouble(),
```

Comment:
so it gives the default lengths of AdSize.banner?

Reply:
yes whatever user selected